src/components/Dialog/Projects.vue

```diff
@@ -59,83 +59,123 @@
+  import { getModule } from "vuex-module-decorators"
+  const appProjects = getModule(Projects)
 
-export default {
-  name: "NewProject",
-  components: {
-    Dialog,
-    Panel,
-    InputText,
-    Button,
-    Dropdown,
-  },
-  props: {
-    projectdialog: { type: Boolean, default: false },
-  },
-  data() {
-    return {
-      type: "",
-      name: "",
-      expanded: "",
-      description: "",
-      icon: "",
-      display: true,
-    }
-  },
-  emits: ["close"],
-  methods: {
-    closeDialog() {
-      this.$emit("close")
+  export default {
+    name: "NewProject",
+    components: {
+      Dialog,
+      Panel,
+      InputText,
+      Button,
+      Dropdown,
     },
-    handleProjectstore() {
-      const data = {
-        type: "project",
-        name: this.name,
-        description: this.des,
-        icon: this.icon,
+    props: {
+      projectdialog: { type: Boolean, default: false },
+    },
+    data() {
+      return {
+        type: "",
+        name: "",
+        expanded: "",
+        description: "",
+        icon: "",
+        display: true,
       }
-      appProjects.addToList(data)
-      this.$emit("close")
     },
-  },
-}
+    emits: ["close"],
+    methods: {
+      closeDialog() {
+        this.$emit("close")
+      },
+      handleProjectstore() {
+        const data = {
+          type: "project",
+          name: this.name,
+          description: this.des,
+          icon: this.icon,
+          queries: {
+            type: "queries",
+            expanded: true,
+            list: [
+              {
+                name: "query ",
+                description: "",
+                type: "query",
+                connection: "connection 2",
+                icon: "connection",
+                query: "",
+                transformer: "transformer 1",
+              },
+            ],
+          },
+          connections: {
+            type: "connections",
+            expanded: true,
+            icon: "connection",
+            list: [
+              {
+                name: "connection ",
+                title: "connection 1",
+                icon: "connection",
+                description: "",
+                type: "connection",
+              },
+            ],
+          },
+
+          transformers: {
+            type: "transformers",
+            list: [
+              {
+                name: "Transformer ",
+                title: "Transformer 1",
+                icon: "connection",
+                description: "",
+                type: "transformer",
+              },
+            ],
+          },
+        }
+        appProjects.addToList(data)
+        this.$emit("close")
+      },
+    },
+  }
 </script>
-<style  lang="scss">
-.dropdown1{
-  width: 100% !important;;
-}
-input {
-  width: 80%;
-}
-.field {
-  display: grid;
-}
-.projects-dialog {
-  height: 100vh;
-  width: 40vw;
-  .p-dropdown{
+<style lang="scss">
+  .dropdown1 {
+    width: 100% !important;
```

Comment:
Do not use !important, unless it is the only option left.

Reply:
this issue have been resolved in ticket #149